{"context":"ExceptionsHandler","level":"error","message":"data.split(...).split is not a function","stack":["TypeError: data.split(...).split is not a function\n    at OauthController.githubCallback (D:\\local_reposiory\\dev-logs\\server\\src\\modules\\oauth\\oauth.controller.ts:25:34)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at D:\\local_reposiory\\dev-logs\\server\\node_modules\\@nestjs\\core\\router\\router-execution-context.js:46:28\n    at D:\\local_reposiory\\dev-logs\\server\\node_modules\\@nestjs\\core\\router\\router-proxy.js:9:17"]}
